diff --git a/middleware/persist.d.ts b/middleware/persist.d.ts
index acb6de52d514261bea23d2e71b3e484165a73d81..a5d4af457c38fec90b21e6b978969042c015014b 100644
--- a/middleware/persist.d.ts
+++ b/middleware/persist.d.ts
@@ -92,8 +92,8 @@ declare type Write<T extends object, U extends object> = Omit<T, keyof U> & U;
 declare type Cast<T, U> = T extends U ? T : U;
 declare type WithPersist<S, A> = S extends {
     getState: () => infer T;
-} ? Write<S, StorePersist<Cast<T, State>, A>> : never;
-interface StorePersist<S extends State, Ps> {
+} ? Write<S, StorePersist<Cast<T, State>, Cast<A, Partial<Cast<T, State>>>>> : never;
+interface StorePersist<S extends State, Ps extends Partial<S>> {
     persist: {
         setOptions: (options: Partial<PersistOptions<S, Ps>>) => void;
         clearStorage: () => void;
